subsys/bluetooth/audio/bap_broadcast_assistant.c

```diff
@@ -605,10 +630,16 @@ int bt_bap_broadcast_assistant_scan_start(struct bt_conn *conn, bool start_scan)
 	int err;
 
 	if (conn == NULL) {
+		LOG_DBG("conn is NULL");
+
```

Comment:
Maybe there should be ENOTCONN instead of EINVAL what do you think? (applies in multiple places)

Reply:
ENOTCONN is used if the state of the `conn` pointer is not `CONNECTED`, so to avoid confusion with that we usually use `EINVAL` if the `conn` pointer is just NULL :)